enn_zoo/enn_zoo/procgen_env/big_fish.py

```diff
@@ -0,0 +1,142 @@
+from typing import Dict, List, Mapping, Tuple
+from enn_zoo.procgen_env.deserializer import ByteBuffer, ProcgenState
+from entity_gym.environment import *
+from procgen import ProcgenGym3Env
+
+
+ENTITY_FEATS = [
+    "x",
+    "y",
+    "vx",
+    "vy",
+    "rx",
+    "ry",
+    "type",
+    "image_type",
+    "image_theme",
+    "render_z",
+    "will_erase",
+    "collides_with_entities",
+    "collision_margin",
+    "rotation",
+    "vrot",
+    "is_reflected",
+    "fire_time",
+    "spawn_time",
+    "life_time",
+    "expire_time",
+    "use_abs_coords",
+    "friction",
+    "smart_step",
+    "avoids_collisions",
+    "auto_erase",
+    "alpha",
+    "health",
+    "theta",
+    "grow_rate",
+    "alpha_decay",
+    "climber_spawn_x",
+]
+
+BIG_FISH_FEATS = ["fish_eaten", "r_inc"]
+EXTRA_FEATS = ["dsize", "bigger"]
+
+
+class BigFish(Environment):
+    def __init__(self, extra_feats: bool = False) -> None:
+        self.env = ProcgenGym3Env(
```

Comment:
I suggest making the `distribution_mode` an argument. `ProcgenGym3Env` uses 
 the hard distribution mode by default: https://github.com/openai/procgen/blob/4b5e5d5806c89303a625386a4a6aae200327fdb5/procgen/env.py#L217, but I'd recommend using the `easy` mode by default instead to save compute.